database/db_migrate.py

```diff
@@ -0,0 +1,16 @@
+import imp
+from migrate.versioning import api
+from database.model import db
+from database.config import SQLALCHEMY_DATABASE_URI, SQLALCHEMY_MIGRATE_REPO
+
+v = api.db_version(SQLALCHEMY_DATABASE_URI, SQLALCHEMY_MIGRATE_REPO)
+migration = SQLALCHEMY_MIGRATE_REPO + ('/versions/%03d_migration.py' % (v+1))
+tmp_module = imp.new_module('old_model')
+old_model= api.create_model(SQLALCHEMY_DATABASE_URI, SQLALCHEMY_MIGRATE_REPO)
+exec(old_model, tmp_module.__dict__)
+script = api.make_update_script_for_model(SQLALCHEMY_DATABASE_URI, SQLALCHEMY_MIGRATE_REPO, tmp_module.meta, db.metadata)
+open(migration, 'wt').write(script)

```

Comment:
You are not closing a file descriptor, use [`with` statement](https://docs.python.org/2/reference/compound_stmts.html#the-with-statement).